open-metadata-implementation/adapters/open-connectors/governance-action-connectors/build.gradle

```diff
@@ -14,7 +14,7 @@ dependencies {
     implementation 'org.slf4j:slf4j-api'
     testImplementation 'org.testng:testng'
     implementation 'com.fasterxml.jackson.core:jackson-annotations'
-
+    implementation 'com.fasterxml.jackson.core:jackson-databind'
```

Comment:
the pom excludes annotations, the maven version doesn't - so they are inconsistent? I presume they should be?